pytorch_toolkit/ote/ote/modules/arg_converters/instance_segmentation.py

```diff
@@ -0,0 +1,64 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from .base import BaseArgConverter
+from ..registry import ARG_CONVERTERS
+
+
+@ARG_CONVERTERS.register_module()
+class InstanceSegmentationArgsConverter(BaseArgConverter):
+    train_update_args_map = {
+        'train_ann_files': 'data.train.ann_file',
```

Comment:
this is also can be taken from mmdetection args converter. You can modify configuration files by modifying data.train section adding Repeat dataset with value=1. It will not change your config.

Reply:
Done.